__tests__/components/ChatBotFooter/ChatBotFooter.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import { expect } from "@jest/globals";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/jest-globals";
+
+import ChatBotFooter from "../../../src/components/ChatBotFooter/ChatBotFooter"
+import { TestChatBotProvider } from "../../__mocks__/TestChatBotContext";
+import AudioButton from "../../../src/components/Buttons/AudioButton/AudioButton";
+import EmojiButton from "../../../src/components/Buttons/EmojiButton/EmojiButton";
```

Comment:
This creates a dependency on the audio and emoji buttons. It'll be great to mock these instead so that the tests are isolated to just the `ChatBotFooter` component. You can reference this [**PR**](https://github.com/tjtanjin/react-chatbotify/pull/173/files).

Reply:
Sure im on it